1-JavaScript/Week3/homework/ex3-tellFortune.js

```diff
@@ -27,33 +27,85 @@ body, this code is now written once only in a separated function.
 -----------------------------------------------------------------------------*/
 const numKids = [
   // TODO add elements here
+  '2',
+  '3',
+  '4',
+  '5',
+  '6',
 ];
 
 const partnerNames = [
   // TODO add elements here
+  'Noah',
+  'Aadam',
+  'Alaa',
+  'Jodi',
+  'Elly',
 ];
 
 const locations = [
   // TODO add elements here
+  'Amsterdam',
+  'Rotterdam',
+  'Zwolle',
+  'Hoogeveen',
+  'Almere',
 ];
 
 const jobTitles = [
   // TODO add elements here
+  'web developer',
+  'electrical engineer',
+  'accountant',
+  'business man',
+  'doctor',
 ];
 
 // This function should take an array as its parameter and return
 // a randomly selected element as its return value.
-function selectRandomly(/* TODO parameter(s) go here */) {
+
```

Comment:
Read the comment carefully:

"This function should **_take an array as its parameter_** and **_return a randomly selected element_** as its return value.".

See how that is different from what you have so far?